### ISSUE TITLE
Controller installation fails in Kubernetes 1.24.6

### ISSUE THREAD

**Issue by irizzant** [csi-addons/kubernetes-csi-addons#273]
Hello

I've tried deploying the controller with the following instructions:
````
kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.5.0/deploy/controller/crds.yaml
kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.5.0/deploy/controller/rbac.yaml
kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.5.0/deploy/controller/setup-controller.yaml
```
but after the controller starts I get the following error:
`
2022-11-16T08:52:34.494Z        INFO    All workers finished    {"controller": "csiaddonsnode", "controllerGroup": "csiaddons.openshift.io", "controllerKind": "CSIAddonsNode"}
2022-11-16T08:52:34.494Z        INFO    All workers finished    {"controller": "persistentvolumeclaim", "controllerGroup": "", "controllerKind": "PersistentVolumeClaim"}
2022-11-16T08:52:34.494Z        INFO    All workers finished    {"controller": "volumereplication", "controllerGroup": "replication.storage.openshift.io", "controllerKind": "VolumeReplication"}
2022-11-16T08:52:34.494Z        INFO    All workers finished    {"controller": "reclaimspacejob", "controllerGroup": "csiaddons.openshift.io", "controllerKind": "ReclaimSpaceJob"}
2022-11-16T08:52:34.494Z        INFO    Stopping and waiting for caches
2022-11-16T08:52:34.495Z        INFO    Stopping and waiting for webhooks
2022-11-16T08:52:34.495Z        INFO    Wait completed, proceeding to shutdown the manager
E1116 08:52:34.495138       1 leaderelection.go:334] error initially creating leader election record: Post "https://10.233.0.1:443/apis/coordination.k8s.io/v1/namespaces/csi-addons-system/leases": context canceled
2022-11-16T08:52:34.495Z        ERROR   setup   problem running manager {"error": "open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directory"}
main.main
        /workspace/go/src/github.com/csi-addons/kubernetes-csi-addons/cmd/manager/main.go:179
runtime.main
        /usr/local/go/src/runtime/proc.go:250

````
I had it previously installed in Kubernetes 1.23 and it was working fine

**Comment by irizzant** [comment]
I can also confirm that the `main` version is working fine, it seems a bug in v0.5.0

**Comment by Rakshith-R** [comment]
Thanks for reporting this issue @irizzant 
This is probably due to dependencies.

I think we can have release `v0.6.0` once we fix https://github.com/csi-addons/kubernetes-csi-addons/issues/236

cc @csi-addons/kubernetes-csi-addons-contributors

**Comment by yati1998** [comment]
> I think we can have release `v0.6.0` once we fix #236
> yeah working on it, we can soon have another release.

**Comment by irizzant** [comment]
Ok thanks, at the moment deploying the main version seems to be a working workaround, I'll wait for v0.6.0 to test again.
